src/client/components/icons/Info.tsx

```diff
@@ -18,7 +18,7 @@ export const InfoIconFilled = (props: IconRootProps) => (
     <path
       fillRule="evenodd"
       clipRule="evenodd"
-      d="M14 7.00002C14 10.866 10.866 14 7.00002 14C3.13402 14 0 10.866 0 7.00002C0 3.13402 3.13402 0 7.00002 0C10.866 0 14 3.13402 14 7.00002ZM6.37503 11V6.00003H7.62503V11H6.37503ZM7.00003 4.50003C7.55232 4.50003 8.00003 4.05232 8.00003 3.50003C8.00003 2.94775 7.55232 2.50003 7.00003 2.50003C6.44775 2.50003 6.00003 2.94775 6.00003 3.50003C6.00003 4.05232 6.44775 4.50003 7.00003 4.50003Z"
+      d="M15 7.99999C15 11.866 11.866 15 8.00002 15C4.13402 15 1 11.866 1 7.99999C1 4.13399 4.13402 0.999969 8.00002 0.999969C11.866 0.999969 15 4.13399 15 7.99999ZM7.37503 12V7H8.62503V12H7.37503ZM8.00003 5.5C8.55232 5.5 9.00003 5.05228 9.00003 4.5C9.00003 3.94772 8.55232 3.5 8.00003 3.5C7.44775 3.5 7.00003 3.94772 7.00003 4.5C7.00003 5.05228 7.44775 5.5 8.00003 5.5Z"
```

Comment:
What is this change?

Reply:
The previous icon had a weird built in margin :/ But I have checked that it looks good in the other place it is used now as well!